shell/shared/shell.js

```diff
@@ -257,6 +257,14 @@ if (Meteor.isClient) {
       return Session.get("selectedApp");
     },
 
+    selectedAppMarketingVersion: function () {
+      var controller = Iron.controller();
+      var appMap = controller.state.get("appMap");
```

Comment:
Hmm, it would be more consistent with the other helpers to do UserActions.find() here.

Otherwise, put appMap into the data object (returned by data()) and access it there, since that's what we do everywhere else. I haven't seen this `controller` approach before and it seems a little confusing to have multiple ways of filling in the template...